roles/css-injector/tasks/main.yml

```diff
@@ -0,0 +1,236 @@
+---
+# Ansible role for changing paths in CSS Injector files.
+# https://github.com/SU-SWS/ansible-playbooks
+# ==================================================================
+#
+# This role runs a number of tasks to transform the database in such a way that
```

Comment:
You mention transformations are in the database, but these appear to all be file changes.  Am I reading the replace tasks wrong?

Reply:
copypasta from `change-paths`